fsharp.extraction.targets

```diff
@@ -17,70 +71,44 @@
                 - FSTAR property is defined
 
             Entry points:
-                - FStarSourceFile items will be used to pass top-level fst files to FSTAR during extraction
+                - Compile items will be used to pass top-level fst files to FSTAR during extraction
                 - FSTAR_FLAGS property will be used to pass non-standard flags to FSTAR during extraction
                 
             Outputs:
                 - Generated *.fs files will be stored in EXTRACTED_PATH and added to Compile item
                 - Generated *.fs files will also be added to the list of files removed by Clean target
+
+            Note: 
+                - Compile item will be completely overriden
         -->
 
         <!-- Make sure the FSTAR_EXTRACTED_PATH directory exists before extraction -->
         <MakeDir Directories="$(FSTAR_EXTRACTED_PATH)" Condition="!Exists('$(FSTAR_EXTRACTED_PATH)')" />
-        <!-- <Exec Command="$(FSTAR_EXE) $(FSTAR_WARN_ERROR) $(FSTAR_EXTRACTION_FLAGS) $(FSTAR_FLAGS) @(FStarSourceFile, ' ')"></Exec> -->
-        <Exec Command="$(FSTAR_EXE) $(FSTAR_WARN_ERROR) $(FSTAR_EXTRACTION_FLAGS) $(FSTAR_FLAGS) %(FStarSourceFile.Identity) --extract &quot;+%(FStarSourceFile.Filename)&quot;"></Exec>
+        
+        <!-- Run extraction for .fst files -->
+        <!-- TODO: What about interfaces? Should we at least run verification for them? -->
```

Comment:
@nikswamy Is running the extraction for all fst files enough? Or should I do something with interface files as well? It seems to work fine, but it's still not entirely clear to me.